Doc/c-api/call.rst

```diff
@@ -140,7 +140,7 @@ Vectorcall Support API
    However, the function ``PyVectorcall_NARGS`` should be used to allow
    for future extensions.
 
-   .. versionadded:: 3.8
+   .. versionadded:: 3.9
```

Comment:
PyVectorcall_NARGS() is defined as a static inline function in Python 3.8.0: https://github.com/python/cpython/blob/v3.8.0/Include/cpython/abstract.h#L75